src/convertToObject.test.js

```diff
@@ -106,23 +88,18 @@ describe('convertToObject should', () => {
 
       aspect-ration: 1;
 
-    `),
-    ).toEqual({
-      'box-sizing': 'border-box',
-      width: '300px',
-      'aspect-ration': '1',
+    `)).toEqual({
+      'box-sizing': 'border-box', width: '300px', 'aspect-ration': '1',
     });
```

Comment:
There's a typo in the property name 'aspect-ration' which should be 'aspect-ratio'. Typographical errors in tests can lead to false positives or negatives, so it's important to ensure that the property names match the expected CSS properties.